src/NetworkFunction/NetworkUseCase.php

```diff
@@ -63,10 +63,32 @@ private function argumentStringToKeyValue( string $argument ): array {
 	}
 
 	private function getVisJsOptions( array $arguments ): array {
-		return array_replace_recursive(
+		$visJsOptions = array_replace_recursive(
 			$this->visJsOptions,
 			json_decode( $arguments['options'] ?? '{}', true ) ?? []
 		);
+		$visJsOptions = $this->resolveImageAliases( $visJsOptions );
+		return $visJsOptions;
+	}
+
+	private function resolveImageAliases( array $options ) : array {
+		foreach ($options as $key => $value) {
+			if (is_array( $value ) ) {
+				$options[$key] = $this->resolveImageAliases( $value );
+			} elseif ($key == "image" && preg_match( '/^%fa[brs]-/', $value ) ) {
+				$code = substr( $value, 3, 1 );
+				if ( $code === 'b' ) {
+					$dir = 'brands';
+				} elseif ( $code === 'r' ) {
+					$dir = 'regular';
+				} else { // $code === 's'
+					$dir = 'solid';
+				}
+				$options[$key] = 'extensions/Network/node_modules/@fortawesome/fontawesome-free/svgs/' .
+					$dir . '/' . substr( $value, 5 ) . '.svg';
+			}
+		}
+		return $options;
```

Comment:
Would be nice to have a test for this. Perhaps best to test on its own, without the rest of the use case. Pure function after all. Not sure though.